packages/nextjs/components/Button.tsx

```diff
@@ -21,11 +23,11 @@ type ButtonAsLink = BaseProps &
 type ButtonProps = ButtonAsButton | ButtonAsLink;
 
 function ButtonComponent(
-  { as = "button", variant, disabled, className, ...props }: ButtonProps,
+  { as = "button", variant, disabled, className, leftIcon: LeftIcon, rightIcon: RightIcon, ...props }: ButtonProps,
```

Comment:
I'm not sure we need this extra type param for `leftIcon` and `rightIcon`. Is there a reason you added it?

Reply:
Yep, the left icon in the mobile toolbar

<img width="229" alt="Screenshot 2023-02-02 at 16 26 54" src="https://user-images.githubusercontent.com/186492/216463644-aa30e334-c657-4f75-a299-1c70c77dfd0d.png">

Reply:
Oh I mean having it renamed to `LeftIcon` and `RightIcon`.

Leaving the prop names as is works the same way, so I was wondering why you added the extra renaming

Reply:
Gotcha, yeah I was thinking to make it look like components prop names, but they are self-descriptive

Reply:
Thanks for the good catch ;), done!